src/Illuminate/Auth/Passwords/PasswordBroker.php

```diff
@@ -201,22 +201,6 @@ public function validateNewPassword(array $credentials)
                 $this->passwordValidator, $credentials) && $password === $confirm;
         }
 
-        return $this->validatePasswordWithDefaults($credentials);
-    }
-
-    /**
-     * Determine if the passwords are valid for the request.
-     *
-     * @param  array  $credentials
-     * @return bool
-     */
-    protected function validatePasswordWithDefaults(array $credentials)
-    {
-        list($password, $confirm) = [
-            $credentials['password'],
-            $credentials['password_confirmation'],
-        ];
-
         return $password === $confirm && mb_strlen($password) >= 6;
     }
```

Comment:
:-1: Having this as separate method would allow anyone to create "extensions" of `PasswordBroker` and potentially  only change `validatePasswordWithDefaults` if anyone want to tweak the logic of the method.